### DISCUSSION TITLE
How to add external JS?

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#8924] [Questions (Nuxt 2)]
When adding css, we do it like this in config. 

```js
// inside export default in NUXT.CONFIG.JS
  css: [
    '@/assets/style/main.scss',
  ],
```

How about external javascript :) How to do it? thanks :)

**Comment by lewebsimple** [comment]
You might want to use the `head` property for that.
See Vue Meta's document on that subject [here](https://vue-meta.nuxtjs.org/api/#script)
